extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -277,24 +279,24 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
       this.element,
       'amp-story-page'
     );
-
     this.tagData_ =
+      shoppingData[pageElement.id] &&
```

Comment:
Could we use optional chaining (`.?`) here instead of a logical operator?

Reply:
yes, we can, sounds good.